mne/io/base.py

```diff
@@ -2058,3 +2059,112 @@ def _check_update_montage(info, montage):
                        "because they are EOG channels use the eog parameter."
                        % str(missing_positions))
                 raise KeyError(err)
+
+
+def read_raw(input_fname, montage=None, eog=[], misc=[], reference=None,
+             preload=False, verbose=None):
+    """Reader function for Raw data
+
+    Note: This assumes the default parameters of a given system. This is
+    largely applicable but if you need to customize a parameter, please use the
+    system-specific read_raw_* function.
+
+
+    Parameters
+    ----------
+    input_fname : str | list of str
+        The path to raw file or a list of the raw files to treat as a Raw
+        instance. For files that have automatically been split, only the
+        name of the first file has to be specified. List of raw files is
+        currently only implemented for '.fif' files.
+    montage : None | str | instance of Montage | list of Montage instances
+        If None and eeg only, sensor locations are (0,0,0).
+        If str, this correspond to the path of a Montage instance. If instance,
+        it can be a Montage or a DigMontage. If a list, it can contain a list
+        of distinct montages: a Montage and DigMontage, not two Montages, not
+        two DigMontages.
+    eog : list or tuple of str
+        Names of channels or list of indices that should be designated
+        EOG channels. Values should correspond to channels listed within
+        the raw file.
+    misc : list or tuple of str
+        Names of channels or list of indices that should be designated
+        MISC channels. Values should correspond to channels listed within
+        the raw file.
+    reference : bool | str
+        If True, add average EEG reference projector (if it's not already
+        present).
+    preload : bool or str (default False)
+        Preload data into memory for data manipulation and faster indexing.
+        If True, the data will be preloaded into memory (fast, requires
+        large amount of memory). If preload is a string, preload is the
+        file name of a memory-mapped file which is used to store the data
+        on the hard drive (slower, requires less memory).
```

Comment:
For `RawFIFF` you need to have the `allow_maxshield` option. Or should we have people use `read_raw_fif` if they need more options? If that's the case, add a Notes section making it clear that not all options are exposed and users might need to use type-specific readers. Also add See also section with the relevant readers.